neo/Consensus/ConsensusContext.cs

```diff
@@ -266,6 +304,11 @@ public ConsensusPayload MakePrepareResponse()
             });
         }
 
+        internal void SetKeyPar(KeyPair key)

```

Comment:
I don't like the methods added for testing. Maybe we should simulate a single-node private network in the test.

Reply:
Any similar test?

Reply:
I have a possible solution, but it not works with all unit test together, because all are using the same Store

Reply:
Lets resolve this store situation, I think its easier.

Reply:
Done, i sign the block without `ConsesusContext`